user-guide/modules/ROOT/pages/boost-history.adoc

```diff
@@ -13,7 +13,9 @@ The story of the Boost pass:[C++] Library began, when Robert Klarer, Beman Dawes
 
 == Early Development
 
-The first few libraries that formed the basis of the Boost pass:[C++] Library were created by a group of dedicated and talented pass:[C++] programmers. Early libraries included the https://www.boost.org/libs/smart_ptr[Smart Pointer] library, which provided support for reference counting and garbage collection, and the https://www.boost.org/libs/regex[Regex] library, which added regular expression support to the language. The Boost pass:[C++] Library quickly gained traction and attracted more contributors who shared a common vision of improving the pass:[C++] language through high-quality, portable, and reusable code.
+The first few libraries that formed the basis of the Boost pass:[C++] Library were created by a group of dedicated and talented pass:[C++] programmers. Early libraries included the https://www.boost.org/libs/smart_ptr[Smart Pointer] library, which provided support for reference counting and garbage collection, and the https://www.boost.org/libs/regex[Regex] library, which added regular expression support to the language. The first official release was in 2000, and contained 24 libraries.
```

Comment:
> https://www.boost.org/libs/smart_ptr[Smart Pointer]

We have a macro for boost libraries now. It's `boost:smart_pointer[]`.

Reply:
@alandefreitas boost:smart_pointer[] gives me an error. Is there a list of these macros somewhere?

The others, such as boost:regex[], work fine.

Reply:
![image](https://github.com/cppalliance/site-docs/assets/5369819/25a46c89-a017-4e0c-9e83-eef7c46e758c)

I just rendered this with boost:smart_ptr[]. You can try shallow cloning a clean version of site-docs and see if that works. It might just be a rebasing issue where new code from the boost extension hasn't been pulled.